models/index.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+exports.selectCategories = () => {
+    return db.query("SELECT * FROM categories;")
+    .then((result) => result.rows)
+    .catch((err) => {
```

Comment:
Similarly I would remove the catch block here as you would be handling this by adding the catch in the controller